lib/elixir/lib/uri.ex

```diff
@@ -324,7 +325,7 @@ defmodule URI do
       "https://elixir-lang.org"
 
   """
-  @spec decode(binary) :: binary
+  @spec decode(binary) :: String.t()
```

Comment:
I think the argument could also be `String.t()` (same for other functions in this module)

Reply:
I am submitting a commit that replaces all `binary` specs with `String.t()`
thank you